packages/server/graphql/types/TeamPromptResponse.ts

```diff
@@ -1,5 +1,6 @@
 import {GraphQLFloat, GraphQLID, GraphQLNonNull, GraphQLObjectType, GraphQLString} from 'graphql'
 import {GQLContext} from '../graphql'
+import {JSONContent} from '@tiptap/react'
```

Comment:
+1 I noticed we sometimes use `@tiptap/react`, sometimes `@tiptap/core`, does it matter?